pycbc/neutron_stars/eos_utils.py

```diff
@@ -0,0 +1,208 @@
+        choice at the moment)
+
+    Returns
+    ----------
+    ns_sequence: numpy.array
+        contains the sequence data in the form NS gravitational
+        mass (in solar masses), NS baryonic mass (in solar
+        masses), NS compactness (dimensionless)
+    max_ns_g_mass: float
+        the maximum NS gravitational mass (in solar masses) in
+        the sequence (this is the mass of the most massive stable
+        NS)
+    """
+    ns_sequence_file = os.path.join(
+        NS_DATA_DIRECTORY, 'equil_{}.dat'.format(eos_name))
+    if eos_name not in NS_SEQUENCES:
+        raise NotImplementedError(
+            f'{eos_name} does not have an implemented NS sequence file! '
+            f'To implement, the file {ns_sequence_file} must exist and '
+            'contain: NS gravitational mass (in solar masses), NS baryonic '
+            'mass (in solar masses), NS compactness (dimensionless)')
+    ns_sequence = np.loadtxt(ns_sequence_file)
+    max_ns_g_mass = max(ns_sequence[:, 0])
+    return (ns_sequence, max_ns_g_mass)
+
+
+def interp_grav_mass_to_baryon_mass(ns_g_mass, ns_sequence):
+    """
+    Determines the baryonic mass of an NS given its gravitational
+    mass and an NS equilibrium sequence (in solar masses).
+
+    Parameters
+    -----------
+    ns_g_mass: float
+        NS gravitational mass (in solar masses)
+    ns_sequence: numpy.array
+        contains the sequence data in the form NS gravitational
+        mass (in solar masses), NS baryonic mass (in solar
+        masses), NS compactness (dimensionless)
+
+    Returns
+    ----------
+    float
+    """
+    x = ns_sequence[:, 0]
+    y = ns_sequence[:, 1]
+    f = interp1d(x, y)
+
+    return f(ns_g_mass)
+
+
+def interp_grav_mass_to_compactness(ns_g_mass, ns_sequence):
+    """
+    Determines the dimensionless compactness parameter of an NS given
+    its gravitational mass and an NS equilibrium sequence.
+
+    Parameters
+    -----------
+    ns_g_mass: float
+        NS gravitational mass (in solar masses)
+    ns_sequence: numpy.array
+        contains the sequence data in the form NS gravitational
+        mass (in solar masses), NS baryonic mass (in solar
+        masses), NS compactness (dimensionless)
+
+    Returns
+    ----------
+    float
+    """
+    x = ns_sequence[:, 0]
+    y = ns_sequence[:, 2]
+    f = interp1d(x, y)
+
+    return f(ns_g_mass)
+
+
+def initialize_eos(ns_mass, eos):
+    """Load an equation of state and return the compactness and baryonic
+    mass for a given neutron star mass
+
+    Parameters
+    ----------
+    ns_mass : {float, array}
+        The mass of the neutron star, in solar masses.
+    eos : str
+        Name of the equation of state.
+
+    Returns
+    -------
+    ns_compactness : float
+        Compactness parameter of the neutron star.
+    ns_b_mass : float
+        Baryonic mass of the neutron star.
+    """
+    if isinstance(ns_mass, numpy.ndarray):
```

Comment:
Sorry, I actually introduced a genuine error here:
```suggestion
    if isinstance(ns_mass, np.ndarray):
```

Reply:
Well, this is trivial enough that I can probably approve my own suggestion.